01/app.js

```diff
@@ -0,0 +1,10 @@
+const user = {
+  firstName: 'Sebastian',
+  lastName: 'Matysiak',
+  sex: 'Male',
+  age: '23',
+};
+
+for (const [key, value] of Object.entries(user)) {
```

Comment:
Uwaga na brak wsparcia przez IE: https://caniuse.com/?search=Object.entries

Reply:
Moze to troche naiwne/ignoranckie podejscie, ale nie za bardzo przejmuje sie IE. Bylem tego swiadomy, kiedy zdecydowalem sie na Object.entries